cmake/dependencies/AMReX.cmake

```diff
@@ -66,7 +65,7 @@ macro(find_amrex)
     else()
         message(STATUS "Searching for pre-installed AMReX ...")
         # https://amrex-codes.github.io/amrex/docs_html/BuildingAMReX.html#importing-amrex-into-your-cmake-project
-        find_package(AMReX 21.02 CONFIG REQUIRED COMPONENTS PIC)
+        find_package(AMReX 21.02 CONFIG REQUIRED COMPONENTS PARTICLES PIC)
```

Comment:
What does this addition do?

Reply:
We can find AMReX in three ways:
- download & compile it (default) or
- pass a source location and compile it or
- find a pre-compiled AMReX.

This option here is the last case. We look for a pre-compiled AMReX build and we make sure that it was compiled with particles enabled.

Reply:
Great, thanks!